doc/ref_arch/kubernetes/chapters/chapter04.rst

```diff
@@ -976,3 +976,216 @@ Additional required components
+      Service :cite:p:`k8s-service-load-balancer`
+    - Reference Model :cite:p:`refmodel` Chapter 5.1.5
+  * - ra2.plat.004
+    - Load balancer and service proxy
+    - If a load balancer is supported it must support workload resource scaling
+    - pas.lb.001 in Reference Model :cite:p:`refmodel` Chapter 5.1.5
+  * - ra2.plat.005
+    - Load balancer and service proxy
+    - If a load balancer is supported it must support resource resiliency
+    - pas.lb.002 in Reference Model :cite:p:`refmodel` Chapter 5.1.5
+  * - ra2.plat.006
+    - Load balancer and service proxy
+    - If a load balancer is supported it must support scaling and resiliency in the local environment
+    - pas.lb.003 in Reference Model :cite:p:`refmodel` Chapter 5.1.5
+  * - ra2.plat.007
+    - Load balancer and service proxy
+    - If a load balancer is supported it must support OSI Level 3/4 load balancing
+    - pas.lb.004 in Reference Model :cite:p:`refmodel` Chapter 5.1.5
+  * - ra2.plat.008
+    - Load balancer and service proxy
+    - If a load balancer is supported it must support round-robin load balancing
+    - pas.lb.005 in Reference Model :cite:p:`refmodel` Chapter 5.1.5
+  * - ra2.plat.009
+    - Load balancer and service proxy
+    - If a load balancer is supported it must create event logs with the appropriate severity levels (catastrophic,
+      critical, and so on)
+    - pas.lb.006 in Reference Model :cite:p:`refmodel` Chapter 5.1.5
+  * - ra2.plat.010
+    - Load balancer and service proxy
+    - If a load balancer is supported it must support monitoring of endpoints
+    - pas.lb.006 in Reference Model :cite:p:`refmodel` Chapter 5.1.5
+  * - ra2.plat.011
+    - Load balancer and service proxy
+    - If a load balancer is supported it must support Direct Server Return (DSR)
+    - pas.lb.006 in Reference Model :cite:p:`refmodel` Chapter 5.1.5
+  * - ra2.plat.012
+    - Load balancer and service proxy
+    - If a load balancer is supported it must support support stateful TCP load balancing
+    - pas.lb.006 in Reference Model :cite:p:`refmodel` Chapter 5.1.5
+  * - ra2.plat.013
+    - Load balancer and service proxy
+    - If a load balancer is supported it must support support UDP load balancing
+    - pas.lb.006 in Reference Model :cite:p:`refmodel` Chapter 5.1.5
+  * - ra2.plat.014
+    - Load balancer and service proxy
+    - If a load balancer is supported it must support load balancing and the correct handling of fragmented packets
+    - pas.lb.006 in Reference Model :cite:p:`refmodel` Chapter 5.1.5
+  * - ra2.plat.015
+    - Service mesh
+    - If a service mesh is supported the service must should support the Service Mesh Interface
+      :cite:p:`sevice-mesh-interface`
+    - Reference Model :cite:p:`refmodel` Chapter 5.1.5
+  * - ra2.plat.016
+    - Security and compliance
+    - The platform may support any open source security and compliance technology
```

Comment:
maybe we should specify what the technology could do?

Reply:
That is correct RM Table 5-7 mentions Calico and cert-manager. Cert-manager you already added and for me it is not clear what was meant with Calico here. I ill ask RM and if no answer received maybe we just remove this.

Reply:
Removed in [a678216](https://github.com/anuket-project/RA2/pull/8/commits/a678216d5934a1d8665272867c13ebe907187f86)